frontend/pages/admin/ManageUsersPage/components/UsersTable/UsersTableConfig.tsx

```
@@ -276,6 +322,37 @@ const generateActionDropdownOptions = (
+    return ["Global", ...names];
+  }
+  return names;
+};
+
+const generateRoleGroups = (
+  teams: ITeam[],
+  globalRole: UserRole | null
+): { role: string; names: string[] }[] => {
+  const groups: { role: string; names: string[] }[] = [];
+  if (globalRole !== null && teams.length > 0) {
+    groups.push({ role: stringUtils.capitalizeRole(globalRole), names: ["Global"] });
```

ESLint

[error] Replace `·role:·stringUtils.capitalizeRole(globalRole),·names:·["Global"]` with `⏎······role:·stringUtils.capitalizeRole(globalRole),⏎······names:·["Global"],⏎···`

(prettier/prettier)